.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -54,7 +54,5 @@ body:
         Thanks for submitting this issue! Are you a contributor? If not, have you thought about it? 
 
         Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
-        We are gauging interest in a potential system in which many companies pledge a little bit of time each to help get more people into these roles. 
-        See [here](https://github.com/argoproj/argo-workflows/issues/12229) for more information. 
-        If you think you or your company may be interested in getting involved, please add a comment to the issue.
+        See [here](https://github.com/argoproj/argo-workflows/blob/main/community/sustainability_effort.md) for more information. 
```

```suggestion
        See [Sustainability Effort](https://github.com/argoproj/argo-workflows/blob/main/community/sustainability_effort.md) for more information. 
```

- Avoid ambiguous "here", [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#links)
  - This also should have previously been "See [#12229](https://github.com/argoproj/argo-workflows/issues/12229)" like the other templates; that seems to have been missed in #12249

---

sorry for the repeat offense! :) I guess this is an ingrained habit for me.

---

darn, I think I just overwrote this and the other fixes. Let me undo the last commit.